operations/c8y_SerialConfiguration

```diff
@@ -1,4 +1,4 @@
 [exec]
-  topic = "c8y/s/dc/modbus"
-  on_message = "3"
-  command = "python3 -m tedge_modbus.operations c8y_SerialConfiguration"
\ No newline at end of file
+  topic = "c8y/devicecontrol/notifications"
+  on_fragment = "c8y_SerialConfiguration"
+  command = "python3 -m tedge_modbus.operations c8y_SerialConfiguration ${.payload.c8y_SerialConfiguration.baudRate} ${.payload.c8y_SerialConfiguration.stopBits} ${.payload.c8y_SerialConfiguration.parity} ${.payload.c8y_SerialConfiguration.dataBits}"
```

Comment:
Couldn't we just pass the c8y_SerialConfiguration contents as json as an argument? That way it wouldn't be sensitive to reordering of the positional arguments

Reply:
Done in 07c899935fef765996e43e755ed7ef17fc6c2ec8